src/components/Messagerie/Filters.jsx

```diff
@@ -198,7 +223,9 @@ const Filters = () => {
               feature={features.messages}
               onSubmit={handleFiltersSubmit}
               values={defaultFilter}
-              onReset={() => setResetFilter(p => p + 1)}
+              onReset={() => setResetFilter(0)}
+              onValuesChange={onValuesChange}
```

Comment:
je pense qu'on peut directement passer la valeur à `true` ici sans faire la comparaison de 2 objets de filtre : 
```suggestion
              onValuesChange={() => setDisabledAction(true)}
```